src/game_engine/unreal/mod.rs

```diff
@@ -56,10 +56,20 @@ impl Module {
         };
 
         let g_world = {
-            const GWORLD: &[(Signature<22>, u8)] = &[(
-                Signature::new("48 8B 05 ?? ?? ?? ?? 48 3B ?? 48 0F 44 ?? 48 89 05 ?? ?? ?? ?? E8"),
-                3,
-            )];
+            const GWORLD: &[(Signature<22>, u8)] = &[
+                (
+                    Signature::new(
+                        "80 7C 24 ?? 00 ?? ?? 48 8B 3D ?? ?? ?? ?? 48 ?? ?? ?? ?? ?? ?? ??",
```

Comment:
The trailing masks seem pointless to me.

Reply:
They are needed for having a Signature<22>, which in turn is needed because I'm looping the sigscan over an array

Reply:
And you can't just make it a `Signature<15>`...?

Reply:
No. Signature<22> and Sifgnature<15> are different types, so you can't make an array of them,
And the second signature needs to be 22.

Reply:
Sounds like incredibly flawed design.